### ISSUE TITLE
contracts with multiple constructor arguments not deploying

### ISSUE THREAD

**Issue by iurimatias** [ethereum/go-ethereum#1284]
```Javascript
// one argument works
// two argument doesn't work

var web3 = require('web3');
var sleep = require('sleep'); //npm install sleep

web3.setProvider(new web3.providers.HttpProvider("http://localhost:8101"));
web3.eth.defaultAccount = web3.eth.accounts[0];

// #####################################
// One argument
// #####################################

source = 'contract SimpleStorage {\n  uint public foo;\n\n  function SimpleStorage(uint y) {\n    foo = y;\n  }\n\n  uint storedData;\n\n  function set(uint x) {\n    storedData = x;\n  }\n  function get() constant returns (uint retVal) {\n    return storedData;\n  }\n}\n'

compiled_contracts = web3.eth.compile.solidity(source)
contract = compiled_contracts.SimpleStorage

contractObject = web3.eth.contract(contract.info.abiDefinition)
// abi definition: [{"constant":false,"inputs":[{"name":"x","type":"uint256"}],"name":"set","outputs":[],"type":"function"},{"constant":true,"inputs":[],"name":"get","outputs":[{"name":"retVal","type":"uint256"}],"type":"function"},{"constant":true,"inputs":[],"name":"foo","outputs":[{"name":"","type":"uint256"}],"type":"function"},{"inputs":[{"name":"y","type":"uint256"}],"type":"constructor"}]


contractAddress = contractObject.new(150, {from: web3.eth.defaultAccount, data: contract.code}).address
console.log("deployed at " + contractAddress);

sleep.sleep(5);
workingContract = contractObject.at(contractAddress);

sleep.sleep(5);
console.log("one argument: foo is " + workingContract.foo().toNumber());
// one argument: foo is 150

// #####################################
// Two arguments
// #####################################

source = 'contract SimpleStorage {\n  uint public foo;\n  uint public bar;\n\n  function SimpleStorage(uint y, uint z) {\n    foo = y;\n    bar = z;\n  }\n\n  uint storedData;\n\n  function set(uint x) {\n    storedData = x;\n  }\n  function get() constant returns (uint retVal) {\n    return storedData;\n  }\n}\n'

compiled_contracts = web3.eth.compile.solidity(source)
contract = compiled_contracts.SimpleStorage

contractObject = web3.eth.contract(contract.info.abiDefinition)
// abi definition: [{"constant":false,"inputs":[{"name":"x","type":"uint256"}],"name":"set","outputs":[],"type":"function"},{"constant":true,"inputs":[],"name":"get","outputs":[{"name":"retVal","type":"uint256"}],"type":"function"},{"constant":true,"inputs":[],"name":"foo","outputs":[{"name":"","type":"uint256"}],"type":"function"},{"constant":true,"inputs":[],"name":"bar","outputs":[{"name":"","type":"uint256"}],"type":"function"},{"inputs":[{"name":"y","type":"uint256"},{"name":"z","type":"uint256"}],"type":"constructor"}]

//contractAddress = contractObject.new(150, 250, {from: web3.eth.defaultAccount, data: contract.code}).address
contractAddress = contractObject.new(150, 250, {from: web3.eth.defaultAccount, data: contract.code, gasLimit: 10000, gasPrice: 10000000000000}).address
console.log("deployed at " + contractAddress);
sleep.sleep(5);

workingContract = contractObject.at(contractAddress);

sleep.sleep(5);
console.log("two arguments: foo is " + workingContract.foo().toNumber());
console.log("two arguments: bar is " + workingContract.bar().toNumber());
// two arguments: foo is 0
// two arguments: bar is 0
````

I experimented with different gas values, the result is the same.

**Comment by iurimatias** [comment]
TLDR; it's gas: not gasLimit:
